v21.2/migrate-from-mysql.md

```diff
@@ -5,6 +5,10 @@ toc: true
 docs_area: migrate
 ---
 
+{{site.data.alerts.callout_success}}
```

Comment:
Is the Migrate from MySQL page in another part of the nav? Because I don't see it, in any of the versions 🤔

Reply:
same for PostgreSQL and Oracle as well.